data/time_series.go

```diff
@@ -82,6 +95,58 @@ func (f *Frame) TimeSeriesSchema() (tsSchema TimeSeriesSchema) {
 	return
 }
 
+// valueToType gets a float64 value and converts it to the specific field type
+// this is useful is fill missing is enabled and fill missing mode is FillMissingValue
+// for converrting the fill missing value (float64) to the field type
+func valueToType(val float64, ftype FieldType) interface{} {
+	switch ftype {
+	case FieldTypeInt8, FieldTypeNullableInt8:
+		return int8(val)
+	case FieldTypeInt16, FieldTypeNullableInt16:
+		return int16(val)
+	case FieldTypeInt32, FieldTypeNullableInt32:
+		return int32(val)
+	case FieldTypeInt64, FieldTypeNullableInt64:
+		return int64(val)
+	case FieldTypeUint8, FieldTypeNullableUint8:
+		return uint8(val)
+	case FieldTypeUint16, FieldTypeNullableUint16:
+		return uint16(val)
+	case FieldTypeUint32, FieldTypeNullableUint32:
+		return uint32(val)
+	case FieldTypeUint64, FieldTypeNullableUint64:
+		return uint64(val)
+	case FieldTypeFloat32, FieldTypeNullableFloat32:
+		return float32(val)
+	case FieldTypeFloat64, FieldTypeNullableFloat64:
+		return val
+	}
+	// if field type is FieldTypeString, FieldTypeNullableString, FieldTypeBool, FieldTypeNullableBool, FieldTypeTime, FieldTypeNullableTime
+	// returns the value unconverted
+	// should return an error instead?
+	return val
+}
+
+func getMissing(fillMissing *LongToWideFillMissing, field *Field, idx int) (interface{}, error) {
+	if fillMissing == nil {
+		return nil, fmt.Errorf("Fill missing is disabled")
```

Comment:
[Error strings should not be capitalized](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).
```suggestion
		return nil, fmt.Errorf("fill missing is disabled")
```

Reply:
@papagian Why resolve this comment? I think it's best to stick with the standard style for error strings, although it's no huge deal.